diff -uNr fedora-ds-1.0.2.orig/nsconfig.mk fedora-ds-1.0.2/nsconfig.mk
--- fedora-ds-1.0.2.orig/nsconfig.mk	2006-04-28 16:43:00.000000000 +0200
+++ fedora-ds-1.0.2/nsconfig.mk	2006-04-28 17:00:34.000000000 +0200
@@ -805,7 +805,7 @@
 else
 ifeq ($(ARCH), SCOOS)
 CC=cc -b elf -KPIC -DSCOOS
-CXX=g++ -b elf -DPRFSTREAMS_BROKEN -I/usr/local/lib/g++-include
+CXX=g++ -b elf -DPRFSTREAMS_BROKEN -I/usr/lib/g++-include
 CCC=$(CXX)
 CPPCMD=/lib/cpp
 ARCH_DEBUG=-g
@@ -1366,7 +1366,9 @@
 MCC_INCLUDE=-I$(BUILD_ROOT)/include \
             -I$(BUILD_ROOT)/include \
             $(NSPR_INCLUDE) $(DBM_INCLUDE) $(SECURITY_INCLUDE) \
-            $(SVRCORE_INCLUDE) -I$(LDAP_INCLUDE) -I$(SASL_INCLUDE)
+            $(SVRCORE_INCLUDE) -I$(LDAP_INCLUDE) -I$(SASL_INCLUDE) \
+	    -I/usr/include -I/usr/include/nspr -I/usr/include/openssl /usr/include/nss
+		
 
 ifeq ($(ARCH), WINNT)
 XP_FLAG=-DXP_WIN32  -DXP_WIN -D_WINDOWS -DXP_PC -DXP_WINNT
