diff -uNr fedora-ds-1.0.2.orig/httpd/autobuild fedora-ds-1.0.2/httpd/autobuild
--- fedora-ds-1.0.2.orig/httpd/autobuild	2005-04-20 00:06:43.000000000 +0200
+++ fedora-ds-1.0.2/httpd/autobuild	2006-04-18 02:41:52.000000000 +0200
@@ -189,34 +189,34 @@
 buildserver() {
 
 (cd ..; ./config $BOMB $PRODUCT $SECCONFIG $OPTIMIZE)
-gmake
+make
 
 if [ $? -ne 0 ]; then
   exit 2
 fi
 
-(cd ../mc-icons; gmake)
+(cd ../mc-icons; make)
 if [ $? -ne 0 ]; then
   exit 2
 fi
 
-(cd ../include; gmake)
+(cd ../include; make)
 if [ $? -ne 0 ]; then
   exit 2
 fi
 
-(cd nsapi; gmake)
+(cd nsapi; make)
 if [ $? -ne 0 ]; then
   exit 2
 fi
 
-(cd extras; gmake)
+(cd extras; make)
 if [ $? -ne 0 ]; then
   exit 2
 fi
 
 if [ "$NOJAVA" != "true" ]; then
-  (cd ../lib/sjava; gmake)
+  (cd ../lib/sjava; make)
   if [ $? -ne 0 ]; then
     exit 2
   fi
@@ -406,7 +406,7 @@
 
 # BONEHEAD
 (cd ..; ./config $BOMB admin $SECCONFIG $OPTIMIZE)
-(cd ../admserv; gmake)
+(cd ../admserv; make)
 if [ $? -ne 0 ]; then
   exit 2
 fi
@@ -443,7 +443,7 @@
 			rm -f $NSPRDIR/*.o
 		fi
 	fi
-	(NS_USE_NATIVE=1; export NS_USE_NATIVE; cd ../batman; gmake MINIRDS_INSTALLDIR="../../httpd/$TARGETUNTAR/plugins" NSAPIDIR="$TARGETUNTAR/nsapi" minirds)
+	(NS_USE_NATIVE=1; export NS_USE_NATIVE; cd ../batman; make MINIRDS_INSTALLDIR="../../httpd/$TARGETUNTAR/plugins" NSAPIDIR="$TARGETUNTAR/nsapi" minirds)
 	if [ $? -ne 0 ]; then
   		exit 2
 	fi
diff -uNr fedora-ds-1.0.2.orig/ldap/clients/Makefile fedora-ds-1.0.2/ldap/clients/Makefile
--- fedora-ds-1.0.2.orig/ldap/clients/Makefile	2005-04-20 00:07:01.000000000 +0200
+++ fedora-ds-1.0.2/ldap/clients/Makefile	2006-04-18 02:41:06.000000000 +0200
@@ -39,7 +39,7 @@
 # END COPYRIGHT BLOCK
 #
 #   Name:
-#   Platform: gmake
+#   Platform: make
 # --------------------------------------------------------------------------
 override BUILD_MODULE=HTTP_ADMIN
 
diff -uNr fedora-ds-1.0.2.orig/ldap/servers/snmp/ntagt/msrvdefs.mak fedora-ds-1.0.2/ldap/servers/snmp/ntagt/msrvdefs.mak
--- fedora-ds-1.0.2.orig/ldap/servers/snmp/ntagt/msrvdefs.mak	2005-04-20 00:07:43.000000000 +0200
+++ fedora-ds-1.0.2/ldap/servers/snmp/ntagt/msrvdefs.mak	2006-04-18 02:40:51.000000000 +0200
@@ -87,7 +87,7 @@
 RM      =   rm -f
 AR      =   ar cr
 MD      =   mkdir -p
-MAKE    =   gmake
+MAKE    =   make
 STRIP   =   strip
 CP      =   cp
 ECHO    =   echo
diff -uNr fedora-ds-1.0.2.orig/nsconfig.mk fedora-ds-1.0.2/nsconfig.mk
--- fedora-ds-1.0.2.orig/nsconfig.mk	2006-02-15 23:53:45.000000000 +0100
+++ fedora-ds-1.0.2/nsconfig.mk	2006-04-18 02:40:04.000000000 +0200
@@ -53,7 +53,7 @@
 
 ABS_ROOT := $(shell cd $(BUILD_ROOT); pwd)
 ABS_ROOT_PARENT := $(shell cd $(ABS_ROOT)/..; pwd)
-MAKE=gmake $(BUILDOPT)
+MAKE=make $(BUILDOPT)
 
 # 7/12/96 Adrian - allow MAKEFLAGS to propagate
 # override MAKEFLAGS := 
@@ -353,7 +353,7 @@
 NSCP_DIST	=  $(BUILD_ROOT)/../dist
 NSCP_DISTDIR          = $(BUILD_ROOT)/../dist/$(NSOBJDIR_NAME)
 
-MAKE=gmake $(BUILDOPT) NO_JAVA=1
+MAKE=make $(BUILDOPT) NO_JAVA=1
 
 export NSPR20=1
 
